cmd/convox/stdcli/stdcli.go

```diff
@@ -264,6 +266,19 @@ func queryExecCommand(bin string, args ...string) ([]byte, error) {
 	return exec.Command(bin, args...).CombinedOutput()
 }
 
+func getDockerBin() string {
```

Comment:
I think this function can go in the `main` package and be used by the various sub commands. Could also use the same method as with the COMPOSE_FILE where you have a `dockerBin` var.